mediorum/server/repair_test.go

```diff
@@ -86,6 +88,8 @@ func TestRepair(t *testing.T) {
 
 		blobs := []Blob{}
 		ss.crud.DB.Where(Blob{Key: cid}).Find(&blobs)
-		assert.True(t, len(blobs) == replicationFactor)
+		assert.Equal(t, replicationFactor, len(blobs))
 	}
+
+	//
```

Comment:
was this gonna be a comment for another test? it's good to see the existing tests btw